packages/components/src/templates/list-with-weights.stories.ts

```diff
@@ -0,0 +1,32 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+export default {
+  tags: ['!dev'],
+  title: 'Templates/List with Weights',
+  parameters: {
+    chromatic: { disableSnapshot: true }
+  }
+};
+
+/**
+ * Bolded text can be used for more emphasis.
```

Comment:
Delete sentence ... we added it to the do’s section

Reply:
Updated